cpp/ql/lib/experimental/quantum/OpenSSL/CtxFlow.qll

```diff
@@ -29,7 +29,19 @@ import semmle.code.cpp.dataflow.new.DataFlow
  * - EVP_PKEY_CTX
  */
 private class CtxType extends Type {
-  CtxType() { this.getUnspecifiedType().stripType().getName().matches("evp_%ctx_%st") }
+  CtxType() {
+    // It is possible for users to use the underlying type of the CTX variables
+    // these have a name matching 'evp_%ctx_%st
+    this.getUnspecifiedType().stripType().getName().matches("evp_%ctx_%st")
```

Comment:
For a C team reviewier, I am curious about necessity of the logic in this charpred (see comment about "build mode none issues" on the lines below.

Reply:
If this is in fact only needed for your test to work, then I would strongly suggest to rewrite the test.